Source/sound.cpp

```diff
@@ -104,7 +104,7 @@ void snd_play_snd(TSnd *pSnd, int lVolume, int lPan)
 
 	tc = GetTickCount();
 	if (tc - pSnd->start_tc < 80) {
-		pSnd->start_tc = GetTickCount();
+		GetTickCount(); // BUGFIX: unnecessary GetTickCount

```

Comment:
:+1: fixed in DevilutionX now also: https://github.com/diasurgical/devilutionX/commit/9496f151cda163b663c925bd71a7105ce921c410